randovania/games/prime1/gui/ui_files/preset_prime_qol.ui

```diff
@@ -62,11 +62,8 @@
         </property>
         <item>
          <spacer name="top_spacer">
-          <property name="orientation">
-           <enum>Qt::Orientation::Vertical</enum>
-          </property>
           <property name="sizeType">
-           <enum>QSizePolicy::Policy::Fixed</enum>
+           <enum>QSizePolicy::Fixed</enum>
```

Comment:
Seems like your Qt Creator is outdated.

Reply:
16.0.1 installed from GitHub. Not sure how to fix

Reply:
if it's updated then leave it like that